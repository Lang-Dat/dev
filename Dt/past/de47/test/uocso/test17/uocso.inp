85
411343366 611427950
660804583 515425031
145937268 298477672
219598716 898830643
706956629 18328571
357589282 202274139
32565413 672380636
623161318 664244997
514809852 379701063
801182218 409439819
260620740 950974038
108933023 791247249
721740361 310944480
133278069 329127229
569720776 601315418
678711225 779221602
918286138 455539216
361054176 566934043
635128686 516242778
679309085 545793226
598070621 537267162
930678037 292477865
771493211 545624071
776036457 671093793
28221141 358640087
102172016 927693825
369958960 238623238
349579901 281882605
458148228 713656263
765394100 192168860
41208794 990170112
32863714 104982631
522875184 52921901
708997387 613998218
820213875 229589369
959743051 328818647
310331550 377107821
5929778 508643970
489769655 175497195
732308527 652594541
144371501 916590561
262823261 189130764
504963583 904758806
79201063 940705875
773719080 239825154
949654771 415186309
824201352 711797687
360280434 688893097
709463212 237386372
916104828 108003251
418635465 662700433
777086221 30269208
62838525 391820638
345341586 25712000
285038286 257349389
30097017 731187029
239966290 321181617
531228307 464233388
146956636 945749120
663023564 578840517
544450236 898081038
509979374 611372971
377816688 15733206
31536871 514364890
382647206 856377415
568550325 224175646
564213537 786966316
55012190 423928321
852379624 547729756
61507711 224459687
619952418 585772509
19721107 637342476
790723138 458191349
961178255 118934331
120975642 198308143
809722846 245404075
48262028 738897237
174817092 447125868
491324573 680598449
331295429 315672765
129921933 464421743
959537526 959035998
484032092 573497319
640158978 465602176
809517337 466495388
