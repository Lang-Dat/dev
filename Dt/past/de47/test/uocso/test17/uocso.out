864
48
768
192
16
32
24
192
384
16
896
96
1536
64
640
192
1152
1536
96
1024
32
16
4
448
64
480
768
64
72
960
768
128
480
16
1728
16
192
384
48
32
8
48
32
384
1920
96
512
64
40
48
64
32
192
1152
512
24
64
144
320
576
128
32
480
64
32
384
72
32
48
16
192
192
512
16
96
48
144
320
8
64
64
288
48
576
12
